shop/src/main/java/ro/msg/learning/shop/models/Customer.java

```diff
@@ -0,0 +1,27 @@
+package ro.msg.learning.shop.models;
+import com.sun.istack.NotNull;
+import lombok.*;
+
+import javax.persistence.*;
+import java.util.List;
+
+@Entity
+@Data
+@Table(name = "customer")
+@Getter @Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@ToString(callSuper = true)
+@EqualsAndHashCode(callSuper = true)
+public class Customer extends BaseEntity{
+
+    private String firstname;
+    private String lastname;
+    private String username;
+    private String password;
+    private String emailaddress;
+
+    @OneToMany(fetch = FetchType.LAZY, mappedBy = "customer")
```

Comment:
As we discussed in the meeting today, it is not needed to have a list here. Please delete it because you can obtain the orders for a Customer from DB.